roles/cis_security/tasks/type-files/ubuntu-18-type.yml

```diff
@@ -1566,7 +1566,7 @@
         replace: "{{ item.replace }}"
       loop:
         - { find: "^max_log_file\\s+=\\s+[^{{ log_file_size }}]", replace: "max_log_file = {{ log_file_size }}" } # 4.1.2.1
-        - { find: "^max_log_file_action\\s+=\\s+((?!{{ max_log_file_action }}).)*$", replace: max_log_file_action = {{ max_log_file_action }} } # 4.1.2.2
+        - { find: "^max_log_file_action\\s+=\\s+(.*)$", replace: "max_log_file_action = {{ max_log_file_action }}" } # 4.1.2.2
```

Comment:
The will always cause it to run not just when it doesn't match

Reply:
got it reverting